### PR TITLE
replaced bookmark icon with a save button and changed "filter" to "save" in the search model

### PR DESCRIPTION
**Pull request by willmoon19**
## Description

This started as a ticket having us change the word "filter" to "save" in the search model. It then came up that it was not very intuitive to save a judge or case with the bookmark icon.

[Trello](https://trello.com/c/eBwSU8Tf)
[Loom](https://www.loom.com/share/ebd846e9205e4b48817f81c073a81304)

Fixes # (issue)

The first fix was changing "filter" to "save" in the search model.
The second fix was to change the bookmark icon into a save button. In order to make it a little more intuitive, I disabled the button when nothing is checked, then when you check a box the button will become enabled and change from a light grey to blue using a ternary and adding another slice of state.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by schandrupatla — APPROVED** [review]
Though it's a simple fix, it's very helpful for the user. I like how the save button is enabled when user selects a check box. But, I would advise using a save icon instead of a "save" label for the button. That would make it neater aesthetically.

**Review by ctscofield — CHANGES_REQUESTED** [review]
You handled your state management very well where it is very easy to read and understand, I might point out a that in the CaseTable.js file you commented out the imports on lines 12 and 13 instead of deleting them and I would probably say to just go ahead and delete those to keep the code base clean. For now will mark as "request changes" until that's been handled but other than that great work!

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ import {
   CloseCircleOutlined,
   CheckCircleOutlined,
 } from '@ant-design/icons';
-import Save from '../../../styles/icons/save.svg';
-import Icon from '@ant-design/icons';
+// import Save from '../../../styles/icons/save.svg';
```
Comment (chrisjs87) [inline src/components/pages/Cases/CaseTable.js:12]: You will definitely want to delete this if we are not going to use them!

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -384,11 +388,13 @@ export default function CaseTable(props) {
...
               <Button
                 className="save-cases-btn"
+                disabled={isDisabled}
                 onClick={() => {
                   bookmarkCases(selectedRowID);
                 }}
               >
-                <Icon component={() => <img src={Save} alt="save icon" />} />
+                Save
+                {/* <Icon component={() => <img src={Save} alt="save icon" />} /> */}
```
Comment (chrisjs87) [inline src/components/pages/Cases/CaseTable.js:397]: You will want to remove this as it is what your code was changed from!

**Review by chrisjs87 — CHANGES_REQUESTED** [review]
Overall a very nice change! I find it much more intuitive than the save icon implemented prior. I also REALLY like how the save button becomes highlighted when a case is checked off! This will indicate to a user that they may interact with the button as soon as it comes into focus. The only things that need changing are the commented code that I put comments next to. Excellent job!

**Review by TrevorDewalt — APPROVED** [review]
The saved icon functionality is much more intuitive than previous set up. Good explanation on your video. In terms of general house keeping, I would delete your old code. I think things are really starting to come together nicely!

**Review by RickyKlusmeier — CHANGES_REQUESTED** [review]
This all looks good to me and does make it more intuitive for the user to know that they are saving the judge or case, as that is what the tab says in the sidebar (keeps with naming conventions). Only thing I would suggest doing is removing what you had commented out from the previous code.

**Review by jenna-anderson — APPROVED** [review]
Excellent, clean solution with the ternary operator and added slice of state. I'm wondering if the save option is needed in the search modal (the first part of the Loom video) - what does that save do?

**Review by prodhc — APPROVED** [review]
Great job with this! The loom video was straightforward and helpful. I know the UX/UI team and I talked about how the save icon needing to be changed to make it more intuitive. Probably outside of this ticket's scope, but it would be nice to see the "filter:" that is present over the checkboxes to be moved, or at least only appears once a filter chip is requested by the user. I could see it possibly confusing some users. I did notice the commented-out code that should be removed, but there are plenty of other people pointing it out, so I feel comfortable having this review set to "Approve".

**Review by mathewmahoneyds20 — APPROVED** [review]
Agree with the majority... The send icon is much more swank

File: src/components/pages/Cases/CaseTable.js (hunk 1 above)
Comment (joshuascan) [reply src/components/pages/Cases/CaseTable.js:12]: Agreed!

**Review by joshuascan — APPROVED** [review]
Your loom video was clear and direct, and the new save button is much more intuitive and user friendly. The only thing I'm unclear on is the functionality of the additional save button within the search filters. Is this performing the same action as the save button on the main page? If so, it might be best to do away with it entirely as it is a little redundant and might cause some confusion for the users.

All in all, great work!

**Review by lucaschatham — APPROVED** [review]
Great stuff. Appreciate the detailed Loom video.

**Review by RickyKlusmeier — APPROVED** [review]

**Review by chrisjs87 — APPROVED** [review]
The changes I requested have been implemented. Great work!

**Review by matthewblackbu — APPROVED** [review]
A very nice QoL change that I think we can all get behind good job will

File: src/components/pages/JudgeTable/JudgeTable.js [hunk 3]
```diff
@@ -305,11 +307,12 @@ export default function JudgeTable(props) {
             tab={
               <Button
                 className="save-cases-btn"
+                disabled={isDisabled}
```
Comment (paulstgermain) [inline src/components/pages/JudgeTable/JudgeTable.js:310]: Smart - I love the way you use state to determine if this button should even be pressed. Subtle way to tell the user some action is required on their part before this option becomes available.

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**